scripts/Interface/ui/CMakeLists.txt

```diff
@@ -24,6 +25,7 @@ add_custom_target(CompileUIUI DEPENDS
   CompileUIInElastic
   CompileUIReflectometer
   CompileUISANS
+  CompileUIDataProcessorInterface
```

Comment:
Another option is to ship the code but make the interface invisible to users. Just let me know what is best. Regarding the name, I am open to suggestions.